### ISSUE TITLE
Is this project no longer open source?

### ISSUE THREAD

**Issue by JaneSmith** [Hotrian/OVRdrop-Public#42]
See the title. I understand it's switched to paid software now, but that doesn't mean it can't still be open source. I'm one of those types that prefers to avoid closed source software as much as possible, so... Any information on this? Thanks.

**Comment by Hotrian** [comment]
Yes, it has been closed since I switched titles. “OVRdrop” itself was never open source.

**Comment by JaneSmith** [comment]
That's disappointing, but thanks for the response. And no plans to open source it I assume? I understand you're likely worried about people stealing it, but that isn't necessarily a big issue. Take Borderless Gaming for example. That's open source on GitHub, but also available for sale on Steam, and many people buy it and recommend it to others.
